### PR TITLE
Use the sbt-echo plugin version for echo as well

### PR DESCRIPTION
**Pull request by havocp**

### CODE REVIEW

File: project/echo.scala [hunk 1]
```diff
@@ -18,7 +18,8 @@ object EchoBuild extends Build {
       settings (LocalTemplateRepo.settings: _*)
       settings(
       Keys.resolvers += typesafeIvyReleases,
-      parallelExecution in GlobalScope := false
+      parallelExecution in GlobalScope := false,
+      version := Dependencies.echoPluginVersion
```
Comment (henrikengstrom) [inline project/echo.scala]: I feel a more descriptive name than ``echoPluginVersion`` might be good? It becomes a bit ambiguous now that it has impact on both Echo and sbt-echo. Wdyt?

**Comment by henrikengstrom** [comment]
LGTM - pending the name comment.

**Comment by havocp** [comment]
Done. also I published 0.1.11 in there, so I just left the commit where it was to show when I published.

**Comment by henrikengstrom** [comment]
LGTM

**Pull request merged**